clueless/views.py

```diff
@@ -190,6 +208,58 @@ def start_game_controller(request):
 	else:
 		logger.error('POST expected, actual ' + request.method)
 
+@login_required
+def join_game_controller(request):
+	"""
+	Adds a player to a game
+	Request should include a game_id and a character_id
+	:param request:
+	:return:
+	"""
+	if request.method == 'POST':
+		if 'character_id' not in request.POST:
+			logger.error('character_id not provided')
+			return redirect('joingame')
+		elif 'game_id' not in request.POST:
+			logger.error('game_id not provided')
+			return redirect('joingame')
+		else:
```

Comment:
should we have an extra check to make sure the game isn't already in progress? ie. if you create a game and i join and then you start it, a third player shouldn't be able to join at that point